aapcs64/aapcs64.rst

```diff
@@ -635,7 +724,7 @@ The base standard defines a machine-level calling standard for the A64 instructi
 Machine Registers
 -----------------
 
-The Arm 64-bit architecture defines two mandatory register banks: a general-purpose register bank which can be used for scalar integer processing and pointer arithmetic; and a SIMD and Floating-Point register bank.
+The Arm 64-bit architecture defines two mandatory register banks: a general-purpose register bank which can be used for scalar integer processing and pointer arithmetic; and a SIMD and Floating-Point register bank. In addition, the architecture defines an optional set of scalable vector registers that overlap the SIMD and Floating-Point register bank, accompanied by a set of scalable predicate registers.
```

Comment:
Not your change, but I can't help my inner language-stickler to comment: That semicolon shouldn't be there. I don't think that's how semicolons are used.

Reply:
That's probably true, but can we fix it separately?  This is quite a big change as it is :-)

Reply:
yup, of course.